CHANGELOG.rst

```diff
@@ -13,18 +13,23 @@ Change Log
 
 Unreleased
 ~~~~~~~~~~
+
+[3.13.1] - 2021-06-08
+~~~~~~~~~~~~~~~~~~~~~
+* If an attempt transitions from `ready_to_submit` back to `started`, the proctoring provider
+  backend function `start_exam_attempt` will not be called.
+
+[3.13.0] - 2021-06-07
+~~~~~~~~~~~~~~~~~~~~~
+* If the Django setting `PROCTORED_EXAM_VIEWABLE_PAST_DUE` is false, exam content will not be viewable past
+  an exam's due date, even if a learner has acknowledged their status.
 * Extend exam attempt API to return exam type and to check if
```

Comment:
These bullet points should have been listed under 3.13.0 when that version was merged/released (they were missed due to PRs being merged around the same time)